controllers/k8ssandra/k8ssandracluster_controller.go

```diff
@@ -137,13 +138,13 @@ func (r *K8ssandraClusterReconciler) reconcile(ctx context.Context, kc *api.K8ss
 
 	kcLogger.Info("All dcs reconciled")
 
-	if recResult := r.reconcileStargateAuthSchema(ctx, kc, actualDcs, kcLogger); recResult.Completed() {
-		return recResult.Output()
-	}
-
-	if recResult := r.reconcileReaperSchema(ctx, kc, actualDcs, kcLogger); recResult.Completed() {
-		return recResult.Output()
-	}
+	//if recResult := r.reconcileStargateAuthSchema(ctx, kc, actualDcs, kcLogger); recResult.Completed() {
+	//	return recResult.Output()
+	//}
+	//
+	//if recResult := r.reconcileReaperSchema(ctx, kc, actualDcs, kcLogger); recResult.Completed() {
+	//	return recResult.Output()
+	//}
 
 	if recResult := r.reconcileStargateAndReaper(ctx, kc, actualDcs, kcLogger); recResult.Completed() {
```

Comment:
The method name is now a bit misleading: it also reconciles telemetry. Just FYI.

Reply:
I am renaming to `afterCassandraReconciled`